README.md

```diff
@@ -212,9 +196,9 @@ class BookingEventConsumer : ProtoEventConsumer<BookingEvent.Builder>() {
 
 </tr>
 <tr>
-<td><pre>tabour.poller.polling-threads</pre></td>
-<td><pre>1</pre></td>
-<td>The number of threads that should poll for new messages. Each of those threads will poll a batch of batchSize messages and then wait for the pollDelay interval until polling the next batch.</td>
+<td><pre>tabour.poller.polling-coroutines</pre></td>
+<td><pre>50</pre></td>
+<td>The number of coroutines that should poll/process new messages. Each of those coroutines will poll a batch of batchSize messages and then wait for the pollDelay interval until polling the next batch.</td>
```

Comment:
btw - do we want this to be as high as 50 by default? if this is to consume ±250 messages per second, that seems quite overkill for most of our apps (and maybe most apps that would use this library).
Another concern - are we sure that e.g. ARI can actually handle this number of messages at once?

Reply:
50 is the max http connections of the sqs client, so setting it to the same amount should not cause issues

Reply:
The default option is the most optimized one, as the client is already providing default 50 connections, as for the overkill part as the coroutines are living on the same thread that won't cause any issue.
ARI concern might be valid but it would be up to us to configure it to work with our needs by setting it for less or more

Reply:
Generally I think the PR is now good code-wise, but I'd still reconsider this one as the default, simply because I think every service will have to change it (most services definitely don't need 250 messages per second, since the rough average of most queues is 0 messages per day - like importing, booking confirmations, booking creations etc). And if most services have to change it, maybe it's not the greatest default value. I'd drop it to something like 5 or 10 tbh

Reply:
Ok that can be changed I will put it as 10